e2e/framework/scheme.go

```diff
@@ -6,13 +6,29 @@ import (
 	addonv1alpha1 "open-cluster-management.io/api/addon/v1alpha1"
 	clusterv1 "open-cluster-management.io/api/cluster/v1"
 	authenticationv1alpha1 "open-cluster-management.io/managed-serviceaccount/api/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-var scheme = runtime.NewScheme()
+var (
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
+)
 
 func init() {
-	clusterv1.AddToScheme(scheme)
-	addonv1alpha1.AddToScheme(scheme)
-	k8sscheme.AddToScheme(scheme)
-	authenticationv1alpha1.AddToScheme(scheme)
+	err := clusterv1.AddToScheme(scheme)
```

Comment:
nit: conventionally we do one liner error assertion by `utilruntime.Must(..)` for the scheme registration

Reply:
ah-- like this? 
```
utilruntime.Must(clientgoscheme.AddToScheme(scheme))
```

Reply:
much prettier! fixed!